examples/pcf8523Alarm/pcf8523Alarm.ino

```diff
@@ -0,0 +1,83 @@
+#include "RTClib.h"
+
+// Change this to reflect the interrupt pin you would like to use
+#define INTERRUPT_PIN 3
+
+RTC_PCF8523 rtc;
+
+bool alarm_triggered = false;
+
+void setAlarmExample();
```

Comment:
Oh weird! I thought I had changed that because I had fixed that before. Git itself was behaving rather strangely while I was trying to make some changes so I wonder if some changes I did weren't being reflected when I pushed to remote. I'll have to give things another review. Thanks for pointing it out. :)